src/applications/claims-status/sass/claims-status.scss

```diff
@@ -6,6 +6,12 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/va-tabs";
 
+@media (min-width: $medium-large-screen) {
+  #claim-letters {
```

Comment:
Is it possible to avoid using id selectors? They have a high specificity and can be difficult to override.
https://design.va.gov/about/developers#implementing-design-work

Reply:
Another consideration is that since these are single property selectors, using a [margin utility](https://design.va.gov/foundation/utilities/margins) instead.

Reply:
But if the selector is necessary, I'd also look into using the [spacing units function](https://design.va.gov/foundation/spacing-units). This allows keeping consistent horizontal/vertical rhythm using multiples of 8px.

Reply:
I’ll look into a different way to accomplish this. I just threw this in there because the page looks weird without some sort of white space since the content doesn’t take much vertical space unless the veteran has more than a few letters. This is a prototype that was put together specifically for the demo today, I planned on polishing it more before moving it out of draft mode